zero/meta-csp/recipes-cspd/csp-handler/files/cspd.h

```diff
@@ -14,6 +14,7 @@
 #define PORT_T (11U) /* for get temperature */
 #define PORT_I (12U) /* for init photo dir */
 #define PORT_C (13U) /* for capture frame */
+#define PORT_J (15U) /* for capture JPEG*/
 #define PORT_F (14U) /* for get frame count */
```

Comment:
Not for this PR, but can we straighten this out? What's the point of having port definitions with alphabet characters that don't align with unordered port numbers?

Reply:
This is because I implemented `PORT_T`, among others, after `PORT_A`.
I wanted to make the `PORT_XX` part more meaningful, like using 'T' for Temperature, but I think that it hsn't been very effective.
I think it's better to change it to defines like `PORT_GET_TEMPERATURE`in a separate PR.